app/src/main/java/org/traccar/manager/DevicesFragment.java

```diff
@@ -53,10 +63,41 @@ public View getView(int position, View convertView, ViewGroup container) {
         }
     }
 
+    @Nullable
+    @Override
+    public View onCreateView(final LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = super.onCreateView(inflater, container, savedInstanceState);
+
+        FrameLayout root = (FrameLayout)view;
+        final Context context = getActivity();
+        FloatingActionButton addButton = new FloatingActionButton(context);
```

Comment:
Use the android provided view creation + an additional floating button to add new entries.